ui/src/components/connections/ConnectionsRowDropdown.tsx

```diff
@@ -26,24 +28,33 @@ const MENU_ITEMS = [
 
 export function ConnectionsRowDropdown({ id }: { id: string }) {
   const navigate = useNavigate();
+
+  const [showModal, setShowModal] = useState<boolean>(false);
+
   const onMenuSelect = (key: string) => {
     if (key === "details") {
       navigate(generatePath(ROUTES.connectionDetails.path, { connectionID: id }));
+    } else if (key === "delete") {
+      setShowModal(true);
     }
   };
 
   return (
-    <Dropdown
-      menu={{
-        items: MENU_ITEMS,
-        onClick: ({ key }) => {
-          onMenuSelect(key);
-        },
-      }}
-    >
-      <Row>
-        <IconDots className="icon-secondary" />
-      </Row>
-    </Dropdown>
+    <>
+      <Dropdown
+        menu={{
+          items: MENU_ITEMS,
+          onClick: ({ key }) => {
+            onMenuSelect(key);
+          },
+        }}
+        overlayStyle={{ zIndex: 999 }}
```

Comment:
Now that there is no rendering overlay with the modal, this isn't needed.

Reply:
The dropdown disappears but is still over the modal while hide
![dropdownoverlay](https://user-images.githubusercontent.com/19190248/230051829-dbda6088-57f1-48d9-abcb-90bd145e0cde.gif)

Reply:
Yeah that's fine, right?

Reply:
The modal background should cover everything